README.md

```diff
@@ -136,7 +136,12 @@ If you're using the Snap version of Firefox, it will be unable to access the hos
 
 ## Chrome
 
-Chrome is currently unsupported, and will not function.
+Accelerated video decoding has been reported to work on Chrome version >= 129 with NVIDIA driver version >= 565.57.01. It might work on slightly older versions of
+Chrome and/or NVIDIA driver, but glitches and bugs have been reported.
+
+These flags should suffice: `--enable-features=AcceleratedVideoDecodeLinuxGL,AcceleratedVideoDecodeLinuxZeroCopyGL,VaapiOnNvidiaGPUs,VaapiIgnoreDriverChecks`. Note that the [direct backend](#direct-backend) must be used.
```

Comment:
I use the following flags - `--use-gl=angle --use-angle=vulkan --enable-features=VaapiOnNvidiaGPUs,AcceleratedVideoDecodeLinuxGL,AcceleratedVideoDecodeLinuxZeroCopyGL,AcceleratedVideoDecode,UseMultiPlaneFormatForHardwareVideo`.

Reply:
and `AcceleratedVideoDecode`?

Reply:
I believe that is redundant with AcceleratedVideoDecodeLinuxGL. I never had that flag enabled.

Reply:
what about `VaapiIgnoreDriverChecks` - is it needed? I don't use it and get `C` in `nvidia-smi`.

Reply:
It is required for some devices, and it can do no harm even if it's not needed - so it's better to include